sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md

```diff
@@ -0,0 +1,67 @@
+
+## Creating a `FormRecognizerClient`
+
+To create a new `FormRecognizerClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if neded, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormRecognizerClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormRecognizerClient(new Uri(endpoint), credential);
+```
+
+## Recognize form content from an Uri
+
+To recognize the content from a given file, use the `StartRecognizeContentFromUri` method. The returned value is a collection of `FormPage` with the respective analizes per page in the document used.
+
+```C# Snippet:FormRecognizerSample3RecognizeContentFromUri
+Response<IReadOnlyList<FormPage>> formPages = await client.StartRecognizeContentFromUri(new Uri(invoiceUri)).WaitForCompletionAsync();
+foreach (FormPage page in formPages.Value)
+{
+    Console.WriteLine($"Form Page {page.PageNumber} has {page.Lines.Count} lines.");
+
+    for (int i = 0; i < page.Lines.Count; i++)
+    {
+        FormLine line = page.Lines[i];
+        Console.WriteLine($"    Line {i} has {line.Words.Count} word {(line.Words.Count > 1 ? "s" : "")}, and text: '{line.Text}'.");
+    }
+
+    for (int i = 0; i < page.Tables.Count; i++)
+    {
+        FormTable table = page.Tables[i];
+        Console.WriteLine($"Table {i} has {table.RowCount} rows and {table.ColumnCount} columns.");
+        foreach (FormTableCell cell in table.Cells)
+        {
+            Console.WriteLine($"    Cell ({cell.RowIndex}, {cell.ColumnIndex}) contains text: '{cell.Text}'.");
+        }
+    }
+}
+```
+
+## Recognize form content from a given file
+
+To recognize the content from a given file, use the `StartRecognizeContent` method. The returned value is a collection of `FormPage` with the respective analysis per page in the document used.
+
+```C# Snippet:FormRecognizerRecognizeFormContentFromFile
+using (FileStream stream = new FileStream(invoiceFilePath, FileMode.Open))
+{
+    Response<IReadOnlyList<FormPage>> formPages = await client.StartRecognizeContent(stream).WaitForCompletionAsync();
+    /*
+     *
+     */
+}
+```
+
+To see the full example source files, see:
+
+* [Recognize form content from Uri](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample3_RecognizeContentFromUri.cs)
+* [Recognize form content from file](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_RecognizeContentFromFile.cs)
```

Comment:
Nit: samples should be numbered in order of complexity ... so if these are just variants of the same functionality, let's number them to be next to each other in the ordering.

Reply:
Should we then use the same {i} as complexity doesn't increase? we did that in TA

Reply:
Yeah, I'm going with that :)